hpx-sys/src/lib.rs

```diff
@@ -19,7 +18,11 @@ pub mod ffi {
         fn terminate();
         fn disconnect() -> i32;
         fn disconnect_with_timeout(shutdown_timeout: f64, localwait: f64) -> i32;
-        //fn stop() -> i32;
+        fn hpx_copy(src: &Vec<i32>, dest: &mut Vec<i32>);
+        fn hpx_copy_n(src: &Vec<i32>, count: usize, dest: &mut Vec<i32>);
+        fn hpx_copy_if(src: &Vec<i32>, dest: &mut Vec<i32>, pred: fn(i32) -> bool);
+        fn hpx_count(vec: &Vec<i32>, value: i32) -> i64;
+        fn hpx_count_if(vec: &Vec<i32>, pred: fn(i32) -> bool) -> i64;
```

Comment:
Is there any way it can be a generic vector type in rust?

Reply:
I've can only do it this [way](https://codeberg.org/pingu-73/demo.git) but for this to work you will have to write separate names for same function with a different data-type and write a rust wrapper to handle these different data-types.
directly you can't do it ig cxx don't support generic vectors

Reply:
Got it, for now let's focus on arithmetic types like I32 only.
Thanks!